tests/suites/0_stateless/06_show/06_0008_show_table_status.sql

```diff
@@ -11,15 +11,23 @@ SHOW TABLE STATUS LIKE 't%';
 SHOW TABLE STATUS LIKE 't2';
 SHOW TABLE STATUS LIKE 't';
 
-SHOW TABLE STATUS WHERE table_name LIKE 't%';
-SHOW TABLE STATUS WHERE table_name = 't%' AND 1 = 0;
-SHOW TABLE STATUS WHERE table_name = 't2' OR 1 = 1;
-SHOW TABLE STATUS WHERE table_name = 't2' AND 1 = 1;
+SHOW TABLE STATUS WHERE name LIKE 't%';
+SHOW TABLE STATUS WHERE name = 't%' AND 1 = 0;
+SHOW TABLE STATUS WHERE name = 't2' OR 1 = 1;
+SHOW TABLE STATUS WHERE name = 't2' AND 1 = 1;
 
 USE default;
-SHOW TABLE STATUS FROM showtabstat WHERE table_name LIKE 't%';
-SHOW TABLE STATUS FROM showtabstat WHERE table_name = 't%' AND 1 = 0;
-SHOW TABLE STATUS FROM showtabstat WHERE table_name = 't2' OR 1 = 1;
-SHOW TABLE STATUS FROM showtabstat WHERE table_name = 't2' AND 1 = 1;
+SHOW TABLE STATUS FROM showtabstat WHERE name LIKE 't%';
```

Comment:
cc @TCeason 
Which one should we use?

`name` or `table_name` ?

Seems `name` is right.

https://www.db-fiddle.com/f/sxCehFV1BHqFbRentu1VRo/0

Reply:
But now we not support alias on select.

Reply:
So may be we should use table_name tmp.

Reply:
@sundy-li @TCeason 

According to  https://dev.mysql.com/doc/refman/8.0/en/show-table-status.html, seems we should polish further the implementation of `show table status` stmt, such that, not only the schema of the resultset adheres to the spec, but also the columns that could be used in the where-clause.

pls take a look at  81544585

Reply:
forget to mention: since the column name is case-sensitive, `name` has be changed to `Name`

Reply:
![image](https://user-images.githubusercontent.com/33082201/162662611-e3c89697-dd92-446b-a626-fb9b0e54a31a.png)

ACTUALLY in MySQL, if the var lower_case_table_names=0;  `name` is the same as `Name`, I think that's ok.

Reply:
> ACTUALLY in MySQL; name is the same as Name, I think that's ok.

roger that